common4j/src/main/com/microsoft/identity/common/java/AuthenticationConstants.java

```diff
@@ -50,6 +66,26 @@
         public static final String CLIENT_REQUEST_ID = "client-request-id";
     }
 
+    /**
+     * Represents the constants value for Active Directory.
+     */
+    @NoArgsConstructor(access = AccessLevel.PRIVATE)
+    public static final class OAuth2 {
```

Comment:
Javadoc says constants for AAD and that's what the class seems to contain, however, the class is named OAuth2. Should it be renamed to something like `AzureActiveDirectory`?